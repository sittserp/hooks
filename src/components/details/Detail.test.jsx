import React from 'react';
import { render, cleanup } from '@testing-library/react';
import Detail from './Detail';

describe('Detail component', () => {
  afterEach(() => cleanup());
  it('renders Detail', () => {
    const { asFragment } = render(<Detail />);
    expect(asFragment()).toMatchSnapshot();
  });
});
