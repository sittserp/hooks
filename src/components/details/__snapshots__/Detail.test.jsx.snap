// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detail component renders Detail 1`] = `
<DocumentFragment>
  <figure>
    <img />
    <figcaption />
    <figcaption />
    <figcaption />
  </figure>
</DocumentFragment>
`;
